### DISCUSSION TITLE
All cyrillic characters are escaped

### DISCUSSION THREAD

**Discussion by Flashwalker** [Taitava/obsidian-shellcommands#318] [Q&A] [answered]
All cyrillic characters of note title and note content are escaped:

Note title:
```
Bla-бла
```

Note content:
```
bla-bla
бла-бла
```

Output:
![image.png](https://user-images.githubusercontent.com/2188215/211368803-f41ccf0e-8504-4e05-a543-15f3f845951d.png)

**Comment by Taitava** [comment, marked as answer]
Hi, thank you for your message! 🙂

The escaping is done for all characters except `a`-`z`, `A`-`Z`, `0`-`9` and `_` (an underscore). For Bash, the escaping `\` characters will disappear, if you leave out the double quotes.

Please let me know if you still have questions. 👍

**Reply by Flashwalker** [reply]
Oh, thanks.
I'm so used to using variables in quotes that I didn't think about it

**Reply by Taitava** [reply]
You are welcome 🙂.

I moved this to _Q&A_.

And seems like I forgot to post a link to [documentation about escaping special characters in variable values](https://publish.obsidian.md/shellcommands/Variables/Escaping+special+characters+in+variable+values).
